content/docs/guides/pwa-amp@es.md

```diff
@@ -0,0 +1,49 @@
+---
+$title: Combina AMP con Aplicaciones Web Progresivas
+$order: 9
+$category: Develop
+toc: true
+components:
+    - youtube
+---
+[TOC]
+
+{{ youtube('Yllbfu3JE2Y', 480, 270, caption='Watch the intro to combining AMP and PWA.') }}
+
+Las Aplicaciones Web Progresivas, en inglés PWA (o Progressive Web Apps) y las páginas AMP trabajan muy bien juntas. De hecho, en muchos casos, se complentan entre ellas de una forma u otra. Aprenda cómo:
+
+1. [Habilite características PWA](/docs/guides/pwa-amp/amp-as-pwa) en su página AMP
```

Comment:
Change links:

- /es/docs/guides/pwa-amp/amp-as-pwa.html
- /es/docs/guides/pwa-amp/amp-to-pwa.html
- /es/docs/guides/pwa-amp/amp-in-pwa.html